articles/quickstart/native/android/_includes/_auth0.md

```diff
@@ -19,10 +19,12 @@ dependencies {
 ```
 
 ::: note
-You can check for the latest version on the repository [Readme](https://github.com/auth0/auth0.android#installation), in [Maven](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22auth0%22%20g%3A%22com.auth0.android%22), or in [JCenter](https://bintray.com/auth0/android/auth0).
+You can check for the latest version on the [repository Readme](https://github.com/auth0/auth0.android#installation), in [Maven](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22auth0%22%20g%3A%22com.auth0.android%22), or in [JCenter](https://bintray.com/auth0/android/auth0).
```

Comment:
@aaguiarz I wasn't sure of what is the latest version mentioned here. Could you help make this note more precise?

Reply:
@awie3di We use `compile 'com.auth0.android:auth0:1.+'` in the articles to use the "latest major" and avoid coming here to update it on every release. If something related to the setup changes then we come and fix it. By adding this "you can check for the latest.." paragraph, people can check which is the "explicit" latest version so they don't use `1.+`

Reply:
@lbalmaceda, is it something that your readers know? Or can we somehow indicate that in the note? Not sure if I understood your comment, if you could help me clarify it for the first-time readers: "You can check for the latest version of X on the ..." Thanks :)

Reply:
Using version `x.+` means use major `X` but latest minor/patch. The problem with fixing it to `x.y` is that each time we release a new minor/patch we need to come here and update the file, otherwise the users will copy paste the version number and keep using the old documented version. This is fine as we follow semver and we don't break stuff in each minor/patch. If that happens, then we will come here and edit the file to say "use at least _this_ new version". By adding a paragraph saying that the explicit last version can be found in the repo url we save from manually editing the file on each release. (we can't automate this yet).